### DISCUSSION TITLE
Updated FlxSpine to use latest HaxeFlixel and spine-hx.

### DISCUSSION THREAD

**Discussion by rhythmpattern** [HaxeFlixel/flixel#3261] [General]
https://github.com/rhythmpattern/spine-flixel-hx.git

Update of some code to run spine-hx on latest HaxeFlixel. Avoiding the offical runtime spine-haxe which uses Ceramic as renderer.
I created this project for others like me who,

    1. Don't want to use Ceramic framework.

    2. Prefer Haxeflixel over heaps for building small indie games.
        a. Android works out of the box
        b. Built in collision detection.
        c. Lot's of haxeflixel samples and easy examples.

Supports v4.0.xx spine files.
    a. Additive blending works.
    b. Clipping is not supported.

Hope this helps someone.
Thank you.

**Comment by Geokureli** [comment]
I'll see if I can get CI to use this fork

**Comment by rhythmpattern** [comment]
I recommend using the latest spine-haxe. Has latest features and this version works without Starling. https://github.com/EsotericSoftware/spine-runtimes/tree/haxe-flixel
